src/App.js

```diff
@@ -4,12 +4,16 @@ import createAuthContext from './lib/createAuthContext'
 const clientId = process.env.REACT_APP_CLIENT_ID || "8cb4904ae5581ecc2b3a1774"
 const clientSecret = process.env.REACT_APP_CLIENT_SECRET || "b683283462070edbac15a8fdab751ada0f501ab48a5f06aa20aee3be24eac9cc"
 const provider = process.env.REACT_APP_PROVIDER || "https://authenticate.u5auth.com"
+const redirectUri = process.env.REACT_RETURN_URI || "https://localhost"
```

Comment:
Why `https` (and not `http`) as default? You normally don't have / use https against localhost.